.devcontainer/devcontainer.json

```
@@ -1,41 +1,28 @@
 {
     "name": "mrc-dev",
     "build": {
-        "context": "..",
-        "dockerfile": "${localWorkspaceFolder}/Dockerfile",
-        "target": "development",
-        "args": {
-            "USERNAME": "${localEnv:USER}",
-            "USER_ID": "${localEnv:UID}"
-        }
+        "dockerfile": "Dockerfile"
     },
-    // todo(252)
-    // new capability with vs code 1.74 (November 2022 Release)
-    // https://github.com/microsoft/vscode-docs/blob/main/remote-release-notes/v1_74.md#gpu-host-requirement
-    // enable this after more testing is conducted
-    // "hostRequirements": {
-    //     "gpu": true
-    // },
+
```

Doing this will let you get rid of `--gpus=all` and `--runtime=nvidia` in the `runArgs`:

```suggestion
"hostRequirements": { "gpu": true },
```